dotrix_core/Cargo.toml

```diff
@@ -20,8 +20,8 @@ version = "0.13.0"
 version = "1.4"
 features = ["derive"]
 
-[dependencies.cgmath]
-version = "0.17"
+[dependencies.dotrix_math]
+path = "../dotrix_math"
```

Comment:
version is missing. It is important, because in other case publication on crates.io will fail